addons/web/static/src/core/emoji_picker/emoji_picker.js

```diff
@@ -274,10 +274,10 @@ export class EmojiPicker extends Component {
             () => [this.searchTerm]
         );
         onWillUnmount(() => {
-            this.navbarResizeObserver.disconnect();
             if (!this.gridRef.el) {
                 return;
             }
+            this.navbarResizeObserver.disconnect();
```

Comment:
I'd personally not trust `gridRef.el` being necessarily present when emojis did not fail.
`Ref.el` may sometimes not work because ref are weird in OWL 2. The guard was there mostly to prevent storing the scroll position when this isn't present.

I would play safe and `this.navbarResizeObserver?.disconnect();` before the `!gridRef.el`, just to make sure that if there's an observer being setup, this is properly disconnected even when ref.el is not there.

I had issues in the past where I expect `ref.el` to work but it doesn't because they were not yet properly processed by OWL. Maybe this was a bug and is fixed, maybe this is still a known limitation / problem of OWL 2.